Sprint1/src/main/java/model/TheatreManager.java

```diff
@@ -70,4 +71,30 @@ private void LoadFromDisk() {
 			e.printStackTrace();
 		}
 	}
+	
+	public Theatre searchTheatreById() {
```

Comment:
Since this is a model class we need to move the logging functions into the ConsoleController and ConsoleView. For now this will do but it needs to be refactored before the end of Sprint 1